components/email_aliases/email_aliases_service.cc

```diff
@@ -0,0 +1,68 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/email_aliases/email_aliases_service.h"
+
+#include <algorithm>
```

Comment:
why do you have these headers? Same with bind.h and string_util.h

Reply:
removed.